src/board.py

```diff
@@ -38,21 +44,39 @@ def get_movables_tiles(board, player_value=0):
     Returns:
         Dictionnary -- A dictionnary with coords in key and instruction in value
     """
-    movable = {}
-    n = 1
+    movable = []
     for x in range(len(board)):
         for y in range(len(board[x])):
             value = board[x][y]
             if is_movable_tile(x, y) and (value == 0 or value == player_value):
-                movable[(x, y)] = n
-                n += 1
+                movable.append((x, y))
 
     return movable
 
 
 def get_coords_from_movables(movables, n):
```

Comment:
The variable n is not enough understandable in my opinion. You could use instead tile_number_to_move (or something else if you have a better idea).